iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/TerminalMockSerDe.java

```diff
@@ -45,6 +45,7 @@ public TerminalMockSerDe() {
                         .put(IidmVersion.V_1_9, ImmutableSortedSet.of("1.9"))
                         .put(IidmVersion.V_1_10, ImmutableSortedSet.of("1.10"))
                         .put(IidmVersion.V_1_11, ImmutableSortedSet.of("1.11"))
+                        .put(IidmVersion.V_1_12, ImmutableSortedSet.of("1.11"))
```

Comment:
Is this a typo or is it wanted (`V_1_12` with `1.11` instead of `1.12`)?

Reply:
1.12